jest.config.js

```diff
@@ -41,10 +41,10 @@ module.exports = {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 68.67,
-      functions: 92.59,
-      lines: 90.2,
-      statements: 90.42,
+      branches: 70.37,
```

Comment:
is there any reasoning behind these numbers?

Reply:
I believe there are semi-automatic rules that bump them up sometimes

Reply:
Correct, it indicates the code coverage and it gets updated every time you run the tests